backend/docs/erd.svg

```diff
@@ -62,7 +62,7 @@
 <text text-anchor="start" x="288" y="-621.63" font-family="Arial Italic" font-size="10.00" fill="#999999">string ∗</text>
 </g>
 <!-- m_ActiveStorage::Blob&#45;&gt;m_ActiveStorage::Attachment -->
-<g id="edge19" class="edge">
+<g id="edge22" class="edge">
```

Comment:
These changes happened after re-migrating with duties ordering, not sure if they are significant, but maybe still worth merging so this file doesn't show up in changes on every setup?

Reply:
schema didn't change. Re-checout this file from master and commit. Then this diff will go away.